apps/frontend/src/app/globals.module.css

```diff
@@ -0,0 +1,36 @@
+.body {
+  font-family: Montserrat, sans-serif;
+  background-color: #7eb397;
+  margin: 0;
+}
+
+.navbar {
```

Comment:
Do we need all of these? Try removing some and check if the navbar still looks and works as before